### ISSUE TITLE
Helm chart: RoleBinding and ClusterRoleBindings bind to the wrong ServiceAccount

### ISSUE THREAD

**Issue by Obirah** [redhat-cop/resource-locker-operator#61]
I'm currently using v1.1.1 of the Helm chart (as a sub-chart in my own chart) and unfortunately, the ServiceAccount references in the `rbac.yaml` of the Helm chart are broken. 

The RoleBinding `resource-locker-operator-leader-election-rolebinding` and ClusterRoleBindings `resource-locker-operator-manager-rolebinding` and `resource-locker-operator-proxy-rolebinding` all bind to the following ServiceAccount:

```
subjects:
- kind: ServiceAccount
  name: controller-manager
  namespace: system
```

To my understanding, the correct subject should look like this:
```
subjects:
- kind: ServiceAccount
  name: resource-locker-operator-controller-manager
  namespace: {{ .Release.Namespace }}
```

**Comment by raffaelespazzoli** [comment]
released 1.1.2, can you check if that fixes the issue?

**Comment by raffaelespazzoli** [comment]
pleas try with `v1.1.3`
